### ISSUE TITLE
Missing session data

### ISSUE THREAD

**Issue by Lukasz93P** [rollbar/rollbar-php-laravel#113]
Hi - https://github.com/rollbar/rollbar-php-laravel/pull/108 I have added one new comment - error which was supposed to be resolved is still there.

**Comment by bxsx** [comment]
Hi @Lukasz93P

Thanks for reporting it.

For clarification, here's an exception you mentioned in #108:
`Attempt to read property "session" on null (Most recent call first) - /app/vendor/rollbar/rollbar-laravel/src/MonologHandler.php line 62`

I believe this error occurs due to breaking changes in Lumen 8.x. I've just merged #106 which addresses this issue. Would you mind trying the #106 PR (or `master` branch) to see if this works for you?

**Comment by Lukasz93P** [comment]
OK @bxsx thanks, give me a few days, I will check if it works for me.

**Comment by Lukasz93P** [comment]
@bxsx yes it works, but with this solution data is not complete - just look at screen.
<img width="1081" alt="Screenshot 2021-11-05 at 22 43 33" src="https://user-images.githubusercontent.com/40272544/140582025-274aaa15-e9cf-4908-a1a8-b66af17a47bd.png">

Why you are so stubborn with not using solution which I have proposed? It solves everything, instead of using something which works you gave me second not complete solution...

**Comment by bxsx** [comment]
Thanks for the info @Lukasz93P 
It seems PR #106 has fixed the reported issue, but now we have a new problem.

Your PR #108 seems to fix this, therefore we are going to re-review it now.
